packages/eslint-plugin/src/rules/emotion-to-compiled/index.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Rule } from 'eslint';
+
+import type { ImportSpecifier, ImportDeclaration } from 'estree';
+
+const hasStyledImport = (node: ImportDeclaration) => node.source.value === '@emotion/styled';
+const hasCoreImport = (node: ImportDeclaration) => node.source.value === '@emotion/core';
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      recommended: true,
```

Comment:
This isn't in the recommended rules though?

Reply:
removed this line